build-base-image.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+BASE_IMAGE_DIR=${1:-base-image}
+CUSTOM_REPO_FILE=${2:-custom.repo}
+
+REPO_FILE_PATH="${BASE_IMAGE_DIR}/${CUSTOM_REPO_FILE}"
+
+[ ! -f "${REPO_FILE_PATH}" ] && { echo "${CUSTOM_REPO_FILE} does not exist!"; exit 1; }
+
+if [[ -s "${REPO_FILE_PATH}" ]]; then
+    sudo podman build --tag ${BASE_IMAGE_DIR} --build-arg TestRepo="${CUSTOM_REPO_FILE}" -f "${BASE_IMAGE_DIR}/Dockerfile"
+else
+    echo "${CUSTOM_REPO_FILE} is empty!"
```

Comment:
I'd put $REPO_FILE_PATH here, so the message contains the actual path tested and in the message on line 10.

Reply:
that's probably better, I'll change it